packages/css-framework/src/fragments/_sidebar.scss

```diff
@@ -0,0 +1,137 @@
+$bar-width: 60px;
+$bar-width-open: 180px;
+$border: 4px;
+
+body.has-sidebar {
+  padding-left: $bar-width;
+}
+
+.rn-sidebar {
+  height: 100%;
+  width: $bar-width;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: color(neutral, 600);
+  color: color(neutral, white);
+}
+
+.rn-sidebar__nav {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.rn-sidebar__nav-link {
+  display: block;
+  text-decoration: none;
+
+  &:hover, &:focus {
+    outline: none;
+  }
+}
+
+.rn-sidebar__nav-icon {
+  text-align: center;
+  display: inline-block;
+  width: $bar-width - $border;
+}
+
+.rn-iconfill {
+  fill: color(neutral, 200);
+}
+
+
```

Comment:
Do we need the extra line breaks?

Reply:
I put them in just to help organise the file to group styles together

Reply:
Helps group the styles to understand the structure